packages/core/src/definitions/styles.ts

```diff
@@ -56,6 +56,13 @@ export const builtInStyles: VariableDefinitions = {
     defaultValue: 'center',
     displayName: 'Horizontal alignment',
   },
+  cfVisibility: {
+    displayName: 'Visibility toggle',
+    type: 'Boolean',
+    group: 'style',
+    defaultValue: false,
+    description: 'To enable visibility toggle in components',
```

Comment:
[optional] (not sure if this is of any value but I propose aligning it with the other descriptions in here)
```suggestion
    description: 'The visibility of the component',
```